main.js

```diff
@@ -41,6 +41,8 @@ const mb = menubar({
 });
 
 mb.on('ready', () => {
+  require('@electron/remote/main').enable(mb.window.webContents);
```

Comment:
Correct me if I'm wrong, but this looks to re-introduce electron/remote, which we're wanting to remove in https://github.com/gitify-app/gitify/issues/650

@brandonweiss also has a draft PR for fixing our auth windows - https://github.com/gitify-app/gitify/pull/654

Reply:
I'm not well-versed in this implementation, and as long as authentication via PAT is possible, it's sufficient for me. Therefore, it's fine to exclude this part from the implementation.